gammapy/stats/fit_statistics.py

```diff
@@ -155,18 +133,40 @@ def wstat(mu_signal, n_on, n_off, alpha):
 
     References
     ----------
-    * Statistics page :ref:`stats`
     * `Habilitation M. de Naurois, p. 141
       <http://inspirehep.net/record/1122589/files/these_short.pdf>`_
     * `XSPEC page on Poisson data with Poisson background
       <http://heasarc.nasa.gov/xanadu/xspec/manual/XSappendixStatistics.html>`_
     """
+    # Note: This is equivalent to what's defined on the XSPEC page under the
+    # following assumptions
+    # t_s * m_i = mu_signal
+    # t_b * m_b = mu_background
+    # t_s / t_b = alpha
 
-    mu_signal = np.asanyarray(mu_signal, dtype=np.float64)
     n_on = np.asanyarray(n_on, dtype=np.float64)
     n_off = np.asanyarray(n_off, dtype=np.float64)
     alpha = np.asanyarray(alpha, dtype=np.float64)
+    mu_signal = np.asanyarray(mu_signal, dtype=np.float64)
+   
+    mu_background = _get_wstat_background(n_on, n_off, alpha, mu_signal)
```

Comment:
Use consistent variable names from the table as much as possible?
I.e. here and in many other places "background" -> "bkg".